lib/utility/init.ts

```diff
@@ -33,24 +33,6 @@ parentPort.on('removeListener', (name: string) => {
   }
 });
 
-const origNewListener = parentPort[Symbol('kNewListener')];
-parentPort[Symbol('kNewListener')] = function (size: number, type: string, ...args: any[]) {
-  if (type === 'message' && (size - 1) === 0) {
-    parentPort.start();
-  }
-  args.unshift(size, type);
-  Reflect.apply(origNewListener, this, args);
-};
-
-const origRemoveListener = parentPort[Symbol('kRemoveListener')];
-parentPort[Symbol('kRemoveListener')] = function (size: number, type: string, ...args: any[]) {
-  if (type === 'message' && size === 0) {
-    parentPort.pause();
-  }
-  args.unshift(size, type);
-  Reflect.apply(origRemoveListener, this, args);
-};
```

Comment:
We are still properly queuing messages, right? Do we have tests for that?

Reply:
Yup, covered in https://github.com/electron/electron/blob/77eab254a6ce264cf0d40cb8b72717eaabecd0b3/spec/api-utility-process-spec.ts#L200-L212